music_assistant/providers/spotify/parsers.py

```diff
@@ -330,3 +331,76 @@ def parse_podcast_episode(
         episode.metadata.preview = episode_obj["audio_preview_url"]
 
     return episode
+
+
+def parse_audiobook(audiobook_obj: dict[str, Any], provider: SpotifyProvider) -> Audiobook:
```

Comment:
As a future improvement to the Spotify provider, we can create TypedDict models for the data coming from the data. Or switch to the spotifyaio library (from Joost) to do the intermediate part